third_party/2/requests/api.pyi

```diff
@@ -1,11 +1,11 @@
 # Stubs for requests.api (Python 3)
 
-from typing import Union
+from typing import Union, Optional
 
 from .models import Response
 
 def request(method: str, url: str, **kwargs) -> Response: ...
-def get(url: Union[str, unicode], **kwargs) -> Response: ...
+def get(url: Union[str, unicode], params: Optional[dict]=None, **kwargs) -> Response: ...
```

Comment:
Actually, re-reading https://docs.python.org/3/library/typing.html the `AnyStr` type doesn't allow type mixing, which this function allows, so I'll need to replace `AnyStr` with `Union[str, unicode]` (for python2 version) and `Union[str, bytes]` (for py3 version)...